app/src/components/v-image.vue

```diff
@@ -1,12 +1,12 @@
 <template>
-	<img ref="imageElement" :src="srcData" v-bind="attrsWithoutSrc" />
+	<v-icon name="image_not_supported" large v-if="imgError" />
```

Comment:
Lets resolve the linter warnings

Reply:
Yes Sure. Linter warnings shows that `v-if` and `v-else` must be placed before other props and attributes. I have updated and pushed.